pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -126,8 +126,13 @@ func fixUserPartOfHyperVAdmins() error {
 	return errReboot
 }
 
+func checkUserPartOfCrcUsers() error {
+	_, _, err := powershell.Execute(fmt.Sprintf("Get-LocalGroupMember -Group 'crc-users' -Member '%s'", username()))
+	return err
+}
```

Comment:
we should mention in the commit logs about which commit removed that check and we are adding back.

Reply:
ahh, i couldn't find the exact commit where it was removed, iirc it was during the time we were adding the msi

Reply:
As far as I can tell, this was https://github.com/crc-org/crc/commit/f98e79c41ea4f752204d2d70d4db705637f9f3d0
(found it using `git log -p pkg/crc/preflight/*win*.go` followed by a search for `Get-LocalGroupMember `. Can be done more directly using  `git log -p -G 'Get-LocalGroupMember'`)